wbooks/src/app/screens/Library/index.tsx

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
-import { FlatList, ListRenderItem } from 'react-native';
+import { SafeAreaView, View, FlatList, ListRenderItem } from 'react-native';
 import { Book } from '@interfaces/Book';
+import { BOOKS_MOCK } from '@constants/mockBooks';
 
 import BookComponent from './components/Book';
+import styles from './styles';
 
-interface Props {
-  books: Book[];
-}
-
-function Library({ books }: Props) {
+function Library() {
   const getKeyExtractor = (item: Book) => `${item.id}`;
 
   const renderItem: ListRenderItem<Book> = ({ item }) => (
     <BookComponent id={item.id} imageUrl={item.imageUrl || ''} title={item.title} author={item.author} />
```

Comment:
`imageUrl={item.imageUrl || ''}` => `imageUrl={item.imageUrl}` you must handle the null case in `BookComponent`

Reply:
Mmm and add `undefined` case too, right?